docker-compose.yml

```diff
@@ -22,7 +22,8 @@ services:
       - DJANGO_DEBUG_TOOLBAR
       - DJANGO_SECRET_KEY=not-secret-in-dev
       - DJANGO_WHITENOISE_AUTOREFRESH=True
-      - TAG_MANAGER_ENABLED=False
+      - TAG_MANAGER_ENABLED=True
+      - TAG_MANAGER_CONTAINER_ID=GTM-NSN42DS
```

Comment:
We don't want values hard coded like `GTM-NSN42DS`, this will come from the environment's variables. @cyber237

Reply:
Okay noted